src/main/java/com/digital/home/service/SalaryService.java

```diff
@@ -0,0 +1,32 @@
+package com.digital.home.service;
+
+import com.digital.home.model.LabourDetails;
+import com.digital.home.model.MaterialDetails;
+import com.digital.home.model.SalaryDetails;
+import com.digital.home.repository.LabourDetailsRepository;
+import com.digital.home.repository.MaterialDetailsRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class SalaryService {
+
+    @Autowired
+    private MaterialDetailsRepository materialDetailsRepository;
+
+    @Autowired
+    private LabourDetailsRepository labourDetailsRepository;
+
```

Comment:
This should be two different methods. Also you need to add two more optional parameters in both the methods as fromDate and toDate. The salary details and material details will be collected for that period only. If these 2 parameters are not passed, you can send all the details for reporting purpose
1. getSalaryDetails
2. getMaterialDetails

Reply:
Added 2 different methods as suggested.